.github/workflows/ci-conda-deployment.yml

```diff
@@ -142,16 +142,16 @@ jobs:
         source ${conda_loc}/etc/profile.d/conda.sh
         conda create -yq -n builder conda-build conda-verify
 
-    - name: macOS 10.14 SDK
+    - name: macOS 11.0 SDK
```

Comment:
One thing I've never understood with this code: we download the macOS SDK in `setup_conda.sh` and also here. Isn't this wasted work? Or am I missing something fundamental, obvious, or both?

Reply:
So, the setup_conda.sh is only used by the conda test workflow (ci-conda-workflow.yml) the other sdk setup is only in the conda package build workflow (ci-conda-deployment.yml).

This is mostly a product of the old Conda package build being in a separate gitlab mirror (then moved here due to changes with GitLab.com). We needed to add an sdk install step (as we used to already have it installed on the gitlab runner) and pulled what was needed. We couldn't use the setup_conda.sh wholesale due to a lot of the other setup existing in the script.

In the future, we should be able to re-write the setup_conda.sh and move some things around to reduce duplication (as that would be nice)